module/TaskManagement/test/integration/TaskManagement/TaskAcceptanceTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+namespace TaskManagement;
+
+use People\Organization;
+use TaskManagement\Entity\Vote;
+use Test\Mailbox;
+use Test\ZFHttpClient;
+
+class TaskAcceptanceTest extends \PHPUnit_Framework_TestCase
+{	
+	protected $client;
+
+    private static $tokens = [
```

Comment:
questo codice inizia ad essere sparso in diversi punti, vale la pensa rifattorizzare secondo te?

Reply:
Fatto, recuperato dalla classe helper